### PR TITLE
Add justification document for TensorFlow21Urllib3Step unit

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: _justifications/tf_21_urllib3.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+Traceback (most recent call last):
+  File "<stdin>", line 1, in <module>
+ImportError: cannot import name 'range'
+```
+
+## Affected packages:
+
+ * ``tensorflow==2.1``
+ * ``urllib3>=1.2<=1.5``
+ * ``six`` in any resovled version
```
Comment (pacospace) [inline _justifications/tf_21_urllib3.md:60]: typo: resolved

**Review by pacospace — COMMENTED** [review]
One small comment, otherwise very nice!!

File: _justifications/tf_21_urllib3.md (hunk 1 above)
Comment (fridex) [reply _justifications/tf_21_urllib3.md]: Fixed, thanks for the review! :+1:

**Review by pacospace — APPROVED** [review]
Thanks :clap: :100:

**Pull request merged**